v2/samples/machinelearningservices/v1api/v1api20210701_workspace.yaml

```diff
@@ -24,18 +24,6 @@ spec:
   # Optional: Save the keys for the storage account into a Kubernetes secret
   operatorSpec:
     secrets:
-      appInsightsInstrumentationKey:
-        name: workspace-secret
```

Comment:
Was there a reason to remove them? 
Rather would suggest commenting them out, so users find it easy to uncomment and export.